colossalai/inference/core/request_handler.py

```diff
@@ -323,6 +327,9 @@ def update_batch_finished(self, batch: BatchBucket, generation_config: Generatio
     def check_unfinished_seqs(self) -> bool:
         return self._has_waiting() or not self.running_list.is_empty()
 
+    def current_requests_in_batch(self) -> int:
```

Comment:
Please use an explicit name. (current requests in which batch, or you want to say "the total requests of XXX")